ES/ESmy001.xml

```diff
@@ -30,324 +30,712 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                              <locus from="17r" to="24"/>
+                              <note>The stucture of the quire is hypothetical.</note>
+                              </item>
+
+                              <item xml:id="q4" n="4">
+                                 <dim unit="leaf">8</dim>
+                                 <locus from="25r" to="32v"/>
+                                 <note>The stucture of the quire is hypothetical.</note>
+                               </item>
+
                               <item xml:id="q5">
                                  <dim unit="leaf">8</dim>
-                                 <locus from="53r" to="60v"/> V(8/fols. 53r-60v) </item>
+                                 <locus from="33r" to="70v"/>
+                                 <note>The stucture of the quire is hypothetical.</note>
+                               </item>
+
                               <item xml:id="q6">
                                  <dim unit="leaf">8</dim>
-                                 <locus from="61r" to="68v"/> VI(8/fols. 61r-68v) </item>
+                                 <locus from="61r" to="68v"/>
+                                </item>
+
                               <item xml:id="q7">
                                  <dim unit="leaf">8</dim>
-                                 <locus from="69r" to="76v"/> VII(8/fols. 69r-76v) </item>
+                                 <locus from="69r" to="76v"/>
+                               </item>
+
                               <item xml:id="q8">
                                  <dim unit="leaf">8</dim>
-                                 <locus from="77r" to="84v"/> VIII(8/fols. 77r-84v) </item>
+                                 <locus from="77r" to="84v"/>
+                               </item>
+
                               <item xml:id="q9">
                                  <dim unit="leaf">8</dim>
-                                 <locus from="85r" to="92v"/> IX(8/fols. 85r-92v) </item>
-                              <item xml:id="q10"> X(...) </item>
-                              <item xml:id="q11">
+                                 <locus from="85r" to="92v"/>
+                               </item>
+
+                              <item xml:id="q10" n="10">
+                              <dim unit="leaf">10</dim>
+                              <locus from="93r" to="101v"/>
+                              </item>
+
+                              <item xml:id="q11" n="11">
                                  <dim unit="leaf">4</dim>
-                                 <locus from="102r" to="105v"/> XI፲፩(4/fols. 102r-105v) </item>
-                              <item xml:id="q12">
+                                 <num value="11">፲፩</num>
+                                 <locus from="102r" to="105v"/>
+                               </item>
+
+                              <item xml:id="q12" n="12">
                                  <dim unit="leaf">5</dim>
```

Comment:
for quires with uneven number of leaves (12, 13, 14) you should specify which leaf is single

Reply:
ok

Reply:
Here, I comment out the collation; it is very hypothetical and should better not be presented in the catalogue.